monai/application/datasets.py

```diff
@@ -120,3 +120,114 @@ def _generate_data_list(self, dataset_dir):
                 raise ValueError("section name can only be: training, validation or test.")
             data.append({"image": image_files_list[i], "label": image_class[i]})
         return data
+
+
+class DecathlonDataset(Randomizable, CacheDataset):
+    """
+    The Dataset to automatically download the data of Decathlon challenge (http://medicaldecathlon.com/)
+    and generate items for training, validation or test. It's based on `CacheDataset` to accelerate
```

Comment:
docstring: CacheDataset -> :py:class:`monai.data.CacheDataset`

Reply:
updated in the latest commit.
Thanks.